unification/chain.json

```diff
@@ -29,49 +29,68 @@
       {
         "denom": "nund"
       }
-    ]
+    ],
+    "lock_duration": {
+      "time": "1814400s"
+    }
   },
   "codebase": {
     "git_repo": "https://github.com/unification-com/mainchain",
-    "recommended_version": "v1.6.3",
+    "recommended_version": "v1.8.2",
     "compatible_versions": [
-      "v1.6.3"
+      "v1.8.2"
     ],
-    "binaries": {
-      "linux/amd64": "https://github.com/unification-com/mainchain/releases/download/v1.6.3/und_v1.6.3_linux_x86_64.tar.gz"
+    "cosmos_sdk_version": "0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
     },
+    "cosmwasm_enabled": false,
+    "ibc_go_version": "5.3.1",
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/unification-com/mainnet/master/latest/genesis.json"
     },
     "versions": [
       {
-        "name": "v1.6.3",
```

Comment:
Instead of writing of v1.6.3 here, can we please just add v1.8.2 as a new version object below?

Reply:
Will do

Reply:
Done. Hopefully nothing is missing